lib/vsc/utils/frozendict.py

```diff
@@ -22,13 +22,13 @@
 """
 import operator
 from functools import reduce
-from future.utils import iteritems
+try:
+    # py 3.10+
+    from collections import Mapping  # noqa
+except ImportError:
+    # < py 3.10
+    from collections.abc import Mapping  # noqa
 
-from vsc.utils.py2vs3 import Mapping
```

Comment:
move this back to some module to handle all this crap somewhere
this will happen more, and it's really ugly to copy/paste these blocks

Reply:
ah, I found out that since python 3.3 you can always do `from collections.abc import Mapping` so no more testing needed. I'll fix.

Reply:
even better